README.md

```diff
@@ -79,7 +69,7 @@ public function up()
 {
     // Add the new location column as nullable
     Schema::table('table', function (Blueprint $table) {
-        $table->point('location')->nullable();
+        $table->geography('location', 'point')->nullable();
```

Comment:
Bu ve yukarıdaki diğer çıkarılan ve değiştirilen alanlar için Laravel 11 öncesi kullanımları sürüm belirterek tutmak daha doğru olur gib.

Reply:
güncelledim